### DISCUSSION TITLE
Do you plan to support GeoJSON Object?

### DISCUSSION THREAD

**Discussion by JimChenWYU** [spatie/laravel-settings#96] [Ideas]
[GeoJSON](https://geojson.org/geojson-spec.html)

I find an [PHP implementation library](https://github.com/jmikola/geojson)

Maybe add an GeoJSONCast Like
```php
class GeoJSONCast implements SettingsCast
{
      public function get($payload)
    {
        return GeoJson::jsonUnserialize($payload);
    }

    public function set($payload)
    {
        return json_encode($payload);
    }
}
```

**Comment by rubenvanassche** [comment]
Hi @JimChenWYU,

I think we want to keep the amount of specific casts in the default package as small as possible. But if you create a package with this cast I can add a link to it in the docs!
